packages/table/style.ts

```diff
@@ -22,3 +22,11 @@ export const tableCss = css`
   font-family: ${fontFamilySansSerif};
   font-weight: normal;
 `;
+
+export const hideScrollbarCss = css`
+  -ms-overflow-style: -ms-autohiding-scrollbar;
```

Comment:
The Jira ticket recommends `-ms-overflow-style: none;`, but when I've tried this in the past with horizontal overflows, IE/Edge users cannot scroll horizontally using a swipes on a trackpad. Can confirm once I have a way to test in IE/Edge